index.html

```diff
@@ -0,0 +1,143 @@
+        </div>
+        <div id="myNav" class="overlay">
+            <a class="closebtn"> &times; </a>
+            <div class="overlay-content">
+                <a href="about.html" class="web-nav-link">About</a>
+                <a href="#program" class="web-nav-link">Program</a>
+                <a href="#devs" class="web-nav-link">Developers</a>
+                <a href="#partner" class="web-nav-link">Sponsor</a>
+                <a href="#" class="web-nav-link ">CN Campaign</a>
+            </div>
+        </div>
+    </header>
+    <br><br><br>
+    <section class="content">
+        <h3 class="caption">"Hello! Devs"</h3>
+        <h1 class="title">
+            Microverse Developers Global Conference 2022
+        </h1>
+        <p class="description">
+            Welcome to the Developers Global Conference, where
+            you meet amazing Software Developers and get mentored
+            by them!
+        </p>
+        <h4 class="date">20-04-2022(WED) ~ 22-04-2022(FRI)</h4>
+        <h5 class="venue">
+            @ San Francisco, California, United States
+        </h5>
+    </section>
+
+    <section id="program">
+        <div>
+            <h1 class="program-title">Main program</h1>
+            <hr>
+            <div class="program-web">
+                <div class="program-item"><i class="fab fa-git icon"></i>
+                    <h4>
+                        Git
+                    </h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                </div>
+                <div class="program-item"><i class="fab fa-github icon"></i>
+                    <h4>
+                        Github
+                    </h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                </div>
+                <div class="program-item"><i class="fab fa-bootstrap icon"></i>
+                    <h4>
+                        Bootstrap
+                    </h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                </div>
+                <div class="program-item"><i class="fa fa-laptop-code icon"></i>
+                    <h4>
+                        Html | Css | Js
+                    </h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                </div>
+                <div class="program-item"><i class="fa fa-server icon"></i>
+                    <h4>
+                        Server
+                    </h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                </div>
+            </div>
+        </div>
+        <a href="#" class="button">Join the Conference</a>
+    </section>
+
+    <h1 class="devs-title">Featured Developers</h1>
+    <hr>
+    <section class="dev_section" id="devs">
+
+    </section>
```

Comment:
Implemented well on the mobile 👍🏽 
- [ ] [OPTIONAL] This could be better on the desktop, currently, the images are far from the text which makes the UI looks kind of unprofessional. Kindly adjust this.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-12 19-31-39](https://user-images.githubusercontent.com/58692590/163030039-7191b2b0-a654-4fed-afa6-49a5affe7104.png) | ![Screenshot from 2022-04-12 19-31-39](https://user-images.githubusercontent.com/58692590/163030116-124f7875-c41d-4208-ae03-24d2f14c139e.png)